snuba/migrations/runner.py

```diff
@@ -45,18 +87,20 @@ def run_migration(self, group: MigrationGroup, migration_id: str) -> None:
         # migrations as in-progress before we execute the operations. However we
         # will need to have some mechanism that allows this to be skipped in certain
         # cases, such as the initial migration that creates the migrations table itself.
-        self._mark_completed(group, migration_id)
+        self._mark_completed(migration_key)
 
         logger.info(f"Finished: {group} {migration_id}")
 
-    def _mark_completed(self, group: MigrationGroup, migration_id: str) -> None:
+    def _mark_completed(self, migration_key: MigrationKey) -> None:
+        status = Status.COMPLETED
+
         statement = f"INSERT INTO {TABLE_NAME} FORMAT JSONEachRow"
         data = [
             {
-                "group": group.value,
-                "migration_id": migration_id,
+                "group": migration_key.group.value,
```

Comment:
I just picked the format to avoid relying on the order of columns. What's the significance of inserting one vs many rows, can't you also do multi rows with `INSERT INTO table Values ((v1a, v1b), (v2a, v2b))`? The only potential limitation I found was that it requires you to provide a value for every column.

Reply:
ok